ovn-tester/ovn_tester.py

```diff
@@ -182,36 +182,77 @@ def configure_tests(yaml, central_node, worker_nodes, global_cfg):
 
 
 def create_nodes(cluster_config, central, workers):
-    mgmt_net = cluster_config.node_net
-    mgmt_ip = mgmt_net.ip + 2
-    internal_net = cluster_config.internal_net
-    external_net = cluster_config.external_net
-    gw_net = cluster_config.gw_net
-    db_containers = (
-        ['ovn-central-1', 'ovn-central-2', 'ovn-central-3']
-        if cluster_config.clustered_db
-        else ['ovn-central']
-    )
+    node_az_conf = [
+        NodeConf(
+            cluster_config.node_net,
+            DualStackSubnet.next(
+                cluster_config.internal_net,
+                i * (cluster_config.n_workers // cluster_config.n_az),
+            ),
+            DualStackSubnet.next(
+                cluster_config.external_net,
+                i * (cluster_config.n_workers // cluster_config.n_az),
+            ),
+            DualStackSubnet.next(
+                cluster_config.gw_net,
+                i * (cluster_config.n_workers // cluster_config.n_az),
+            ),
+        )
+        for i in range(cluster_config.n_az)
+    ]
+
+    db_containers = [
+        [
+            f'ovn-central-az{i+1}-1',
```

Comment:
Nit: Why not start from index 0?

Reply:
@dceara it is to be consistent with ovn-fake-multinode where ovn-chassis starts from 1